app/helpers/auth_source_ldaps_helper.rb

```diff
@@ -3,4 +3,7 @@ def on_the_fly?(authsource)
     return false if authsource.new_record?
     authsource.onthefly_register?
   end
```

Comment:
Rubocop is complaining about a lack of an empty line between methods here, it's the only thing making this PR :x: